### ISSUE TITLE
run juicefs stats failed with invalid value

### ISSUE THREAD

**Issue by diluga** [juicedata/juicefs#805]
```
juicefs version 0.17-dev (2021-08-31 9f190b50)
```
after run `juicefs stats ~/jfs` for a long time,it will failed with `invalid value`

```

------usage------ ----------fuse--------- ----meta--- -blockcache ---object--
 cpu   mem   buf | ops   lat   read write| ops   lat | read write| get   put
67.7%  335M    0 | 177  5.91     0   125K| 125  16.0 |   0   109K|   0   109K
66.8%  335M    0 |  10  0.47     0     0 |  16  61.4 |   0     0 |   0     0
68.6%  335M    0 |   0     0     0     0 |   8   139 |   0     0 |   0     0
71.2%  335M 4096B|   0     0     0     0 |   9   110 |   0     0 |   0     0
94.5%  335M    9M| 437  9.81    16K   63K| 396  2.97 |6936K 5106K|6776K   30K
12.5%  335M 9488K|  46  22.2     0     0 |  46  22.2 |   0   936K| 924K   12K
18.8%  335M 5388K|  11  87.1     0  3919B|  10   242 |   0  3919B|   0  3919B
88.7%  335M   14M| 312  3.70    16K   68K| 281  4.48 |7684K   57K|   0    57K
76.0%  335M 4372K| 129  6.33    16K   77K| 100  10.5 | 392K   64K|   0    64K
81.8%  335M 4568K| 203  5.94  8192B   88K| 164  9.00 | 164K   69K|   0    69K
81.0%  335M 4168K| 291  3.34    12K  136K| 236  4.16 |8192B   89K|   0    89K
79.9%  335M 8260K| 119  9.91    12K   17K| 114  10.5 |4100K   20K|   0    20K
72.8%  335M 4160K| 169  0.80     0   112K| 131  7.91 |   0    89K|   0    89K
 123%  335M 5184K| 336  4.99  8192B  119K| 286  4.87 |1892K 3735K|3800K  103K
97.1%  335M 9280K| 239  5.91  4096B  134K| 190  7.02 |2576K  788K| 696K   92K
90.1%  335M   10M| 157  5.72  8192B   38K| 146  7.55 | 696K 1295K|1969K   35K
80.4%  335M 5184K| 144  4.89     0   111K| 106  12.4 |1800K  964K| 890K   84K
 153%  335M   13M| 316  5.03  8192B  144K| 263  6.38 | 916K 7227K|9580K  147K
 121%  335M   13M|   0     0     0     0 |   8   157 |2484K 5788K|6115K   24K
80.2%  335M   14M| 165  9.33    12K   50K| 137  2.97 | 292K  432K| 436K   32K
77.7%  335M 5768K|  58  29.5  8192B   74K|  30  60.7 |  60K   92K|  80K 8720B
66.1%  335M  260K|  71  9.22     0    20K|  70  23.3 |8192B   76K|   0    80K
2021/09/08 14:33:40.584106 juicefs[35398] <FATAL>: invalid value -14368768.000000
```

**Comment by SandyXSD** [comment]
@diluga Can you please check if the JuiceFS process was killed?

**Comment by SandyXSD** [comment]
Maybe fixed by #806
